setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 setup(name='maxfw',
-      version='1.0.0',
+      version='1.1.0',
```

Comment:
I think we should hold off on a version bump until the PRs addressing the current issues are merged.

Reply:
Which current issues? This is the only open PR. And the reason for the version bump is these changes a version bump level and would need to be pushed out to support the object detector since it's currently reverted and missing documented features

Reply:
The 2 most recent issues here: https://github.com/IBM/MAX-Framework/issues

Reply:
I see, normally I would agree to wait on a version release for those since they'd also require a version release, but given this fix returns expected functionality to the Object Detector that it has lost makes getting this released a high priority.

Reply:
Maybe we can bump it up to 1.0.1 then?

Reply:
I choose 1.1 based on our version documentation. Similarly when I finish fixing Docker I'll be changing the releases from 1.0.1 that Brendan made to 1.1.0 to match the doc

Reply:
https://github.ibm.com/CODAIT/MAX-development-guide/blob/master/VERSIONING.md

Reply:
Would it make more sense to have a 1.1.0-rc? At this rate we'll be bumping the version for every PR

Reply:
We can do rc's if we want, but I think they're a bit overkill for a project with so few devs. And normally I would update release versions in their own PR after release requirements is met, but in the case of this PR it alone meets a release requirement so I just included it. According to our release requirements minor release should wait until all PRs with minor release level changes are merged, but in the case of this PR, this restores lost functionality from a "previous" release and should be release ASAP.

Reply:
`1.1.0` makes sense - I think any update to image utils is likely to take a little time so let's get this out now.